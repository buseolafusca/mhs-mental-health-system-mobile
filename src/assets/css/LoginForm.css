.form-group {
    padding: 1.5rem;
    margin-right: 0;
    margin-bottom: 0;
    margin-left: 0;
    border-width: .2rem;
}

.loginform {
    width: 30%;
    margin: 0 auto;
    margin-top: 5%;
}

.label-title {
    color: dimgray;
    padding: 1.5rem;
    margin-right: 0;
    margin-bottom: 0;
    margin-left: 5%;
    border-width: .2rem;
    font-size: 3vw;
    font-weight: bolder;
    text-shadow: 1px 2px gainsboro;
}

.label-subtitle {
    font-size: 1vw;
    position: relative;
}

#loginbutton {
    width: 70%;
    margin: 0 auto;
    left: 15%;
    margin-top: 10%;
    /* color: #fff;
    background-color: #9c27b0;
    border-color: #9c27b0;
    box-shadow: 0 2px 2px 0 rgba(156, 39, 176, .14), 0 3px 1px -2px rgba(156, 39, 176, .2), 0 1px 5px 0 rgba(156, 39, 176, .12); */
}

.btn:hover {
    /* cursor: pointer;
    background-color: #9c27b0;
    border-color: #9c27b0;
    transition: box-shadow .2s cubic-bezier(.4, 0, 1, 1), background-color .2s cubic-bezier(.4, 0, .2, 1); */
}

.btn {
    /* position: relative;
    padding: 12px 30px;
    margin: .3125rem 1px;
    font-size: 1.15rem;
    font-weight: 400;
    line-height: 1.428571;
    text-decoration: none;
    text-transform: uppercase;
    border: 0;
    border-radius: .2rem;
    transition: box-shadow .2s cubic-bezier(.4, 0, 1, 1), background-color .2s cubic-bezier(.4, 0, .2, 1);
    will-change: box-shadow, transform; */
}

.div-login {
    margin: 0 auto;
    text-align: center;
}

;