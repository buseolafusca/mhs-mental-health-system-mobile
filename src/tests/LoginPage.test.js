import React from 'react'
import { shallow } from 'enzyme'
import { Provider } from "react-redux"
import configureMockStore from "redux-mock-store"
import LoginForm from '../layouts/LoginPage'

const mockStore = configureMockStore();
const store = mockStore({});

describe('Login page', () => {
  it('should render correctly in mode', () => {
    const component = shallow(<Provider store={store}>
                    <LoginForm />
                </Provider>)

    expect(component).toMatchSnapshot()

  })

})

