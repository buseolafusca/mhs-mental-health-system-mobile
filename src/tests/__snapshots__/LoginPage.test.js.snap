// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login page should render correctly 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "clear": [Function],
          "get": [Function],
          "notify": [Function],
          "subscribe": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": [Function],
      },
    }
  }
>
  <LoginForm
    match={
      Object {
        "isExact": true,
        "params": Object {
          "email": "aa",
          "password": "aa",
        },
        "path": "",
        "url": "",
      }
    }
    password={Object {}}
    username={Object {}}
  />
</ContextProvider>
`;

exports[`Login page should render correctly in mode 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "clear": [Function],
          "get": [Function],
          "notify": [Function],
          "subscribe": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": [Function],
      },
    }
  }
>
  <Connect(LoginForm) />
</ContextProvider>
`;

exports[`Login page should render correctly in mode 2`] = `
<div
  className="landing-page-container"
>
  <NHSHeader />
  <form
    action="javascript:void(0)"
    className="login-form"
    noValidate={true}
    onSubmit={[Function]}
  >
    <div
      className="input-container"
    >
      <div
        class="row"
      >
        <div
          class="block"
        >
          <input
            autoComplete="off"
            maxLength={40}
            onChange={[Function]}
            placeholder="Email"
            type="email"
          />
          <ErrorText />
        </div>
        <div
          class="block2"
        >
          <span
            className="glyphicon glyphicon-envelope "
          />
        </div>
      </div>
    </div>
    <div
      className="input-container"
    >
      <div
        class="row"
      >
        <div
          class="block"
        >
          <input
            autoComplete="off"
            maxLength={40}
            name="Password"
            onChange={[Function]}
            placeholder="Password"
            type="password"
          />
          <ErrorText />
        </div>
        <div
          class="block2"
        >
          <span
            className="glyphicon glyphicon-lock"
          />
        </div>
      </div>
    </div>
    <div
      style={
        Object {
          "content": " ",
          "marginLeft": -15,
          "marginRight": -15,
        }
      }
    >
      <Link
        className="btn btn-link"
        to="/register"
      >
        Register
      </Link>
      <div
        className="button-container"
      >
        <button
          className="btnSubmit"
          type="submit"
        >
          Sign In
        </button>
      </div>
    </div>
  </form>
  <NHSFooter />
</div>
`;
